autogen/agentchat/contrib/multimodal_conversable_agent.py

```diff
@@ -64,44 +60,21 @@ def update_system_message(self, system_message: Union[Dict, List, str]):
         self._oai_system_message[0]["role"] = "system"
 
     @staticmethod
-    def _message_to_dict(message: Union[Dict, List, str]):
-        """Convert a message to a dictionary.
+    def _message_to_dict(message: Union[Dict, List, str]) -> Dict:
```

Comment:
It seems like _message_to_dict will fail silently if the message is not of [Dict, List, str] -- silent fails make it a little hard to debug in the future. In that case, shall we assert that the input is of these three types?

Reply:
There is a "raise" clause at the end of the function, which will raise an error if not one of the three types.

Reply:
Yes, my bad, sorry!